var/spack/repos/builtin/packages/sim-model/package.py

```diff
@@ -69,14 +69,12 @@ def __build_mods_coreneuron(self, mods_location, link_flag, include_flag):
         spec = self.spec
         assert os.path.isdir(mods_location), mods_location
         include_flag += ' -I%s' % (spec['coreneuron'].prefix.include)
-        if spec.satisfies('^coreneuron@0:0.16'):
-            which('nrnivmodl-core')(
-                '-i', include_flag, '-l', link_flag, '-n', self.mech_name,
-                '-v', str(spec.version), '-c', mods_location)
-        else:
-            which('nrnivmodl-core')('-i', include_flag, '-l', link_flag, mods_location)
-        output_dir = spec.architecture.target
-        expected_name = "libcorenrnmech"
+        nrnivmodl_params = ['-i', include_flag,
+                            '-l', link_flag,
+                            '-n', self.mech_name]
```

Comment:
If I am not mistaken, this won't work with `^coreneuron@develop` because `-n` option is removed in coreneuron.

Reply:
I think it would be nice to bring it back, to avoid manually renaming the libs after they are created, otherwise Spack recipe complicates a lot

Reply:
Agree. This is much simpler and cleaner. No reason to break the way we call `nrnivmodl-core` for different versions.